modules/bgsegm/include/opencv2/bgsegm.hpp

```diff
@@ -240,6 +240,58 @@ createBackgroundSubtractorCNT(int minPixelStability = 15,
                               int maxPixelStability = 15*60,
                               bool isParallel = true);
 
+enum LSBPCameraMotionCompensation {
+    LSBP_CAMERA_MOTION_COMPENSATION_NONE = 0,
+    LSBP_CAMERA_MOTION_COMPENSATION_LK
+};
+
+class CV_EXPORTS_W BackgroundSubtractorLSBP : public BackgroundSubtractor
```

Comment:
Since the algorithm uses random number generators inside, there should be a way of providing custom generator or setting parameters of built-in generator for testing purposes (to get reproducible results).